src/frontend/search_page/components/SearchPageButton.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export interface ButtonProps {
```

Comment:
I think the SearchPageButton will only have `Add` and `Chat` as text, so we won't use this props anywhere else? And I think it might be better to setup two Classed for `add `and `chat` buttons or use a Enum. We might use a switch for differnet actions on different buttons in the future which will depend on this `text`. It is better to depend on Enums than strings.

Reply:
I was thinking that we can put this interface into `shared` and all the buttons can use this props. Might need to add more properties into it when we add functionalities. Do you mean that I can create a text enum for all the possible text that we would put on a button?